microreactors/gcmr/assembly/RIA_steady_state/BISON.i

```diff
@@ -0,0 +1,464 @@
+TsInit = 1150.0 # Solid initial temperature
+Tcin = 1150.0 # Coolant initial temperature
+radiusTransfer = 0.015 # r + 0.009. Extends past the first mesh cell surrounding the coolant channel.
+coolant_full_points_filename = ../channel_positions/coolant_full_points.txt # File containing the inlet position of your coolant channels
+coolant_half_points_filename = ../channel_positions/coolant_half_points.txt # File containing the inlet position of your coolant channels
+
+[GlobalParams]
+  flux_conversion_factor = 1
+[]
+
+[Problem]
+  register_objects_from = 'BisonApp'
+  #library_path = '/beegfs1/software/NEAMS_microreactor/projects_super_mar22/bison/lib'
```

Comment:
```suggestion
```

We should remove this library path. This input uses both SAM and BISON objects, maybe a good showcase for dynamic linking.